recipe/meta.yaml

```diff
@@ -48,23 +48,23 @@ requirements:
     - ipython
     - jupyter
     - livvkit 3.2.0
-    - mache 1.32.0
+    - mache 2.0.0rc2
```

Comment:
I was wondering, "why is it complaining?  Didn't we build this one?"  And, yes, we did.  But we didn't add its label to the list of source channels.

Reply:
Ah, rerender, of course!